examples/fibo.vy

```diff
@@ -0,0 +1,4 @@
+1…1{~+…}
```

Comment:
This is improper Fibonacci. It only prins one `1`.

Maybe `1{…~+}` or `0 1{…~+}`? Or `1 0{…~+}` to print a zero too.